src/api/app/models/concerns/staging_project.rb

```diff
@@ -142,6 +169,7 @@ def cache_problems
   end
 
   def state
+    return :merging if Delayed::Job.where("handler LIKE '%StagingMergeProjectJob%'").exists?
```

Comment:
This makes all staging projects for all staging workflows :merging if one of them is. Not an option

Reply:
I know. I just wanted something that remotely works. So I could try out the stuff around this and write some tests.

Reply:
`handler` is just the job model serialized as haml. You can basically do 

```
job = YAML::load(blah.handler).job_data
```

for a hash with the job in it, that would contain the specific project. I guess this could also go neatly into the concern.

Reply:
I don't think we can do this, since we need to find the correct delayed job first. What we probably could do is to save the DJ identifier and use it here to check if the job is finished, aka gone, or not.

But what we do right now (checking the job class and the project id) should also be fine.